tests/templates/kuttl/smoke/10-assert.yaml

```diff
@@ -386,164 +443,13 @@ data:
-metadata:
-  labels:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/managed-by: opensearch.stackable.tech_opensearchcluster
-    app.kubernetes.io/name: opensearch
-    app.kubernetes.io/role-group: data
-    app.kubernetes.io/version: 3.0.0
-    stackable.tech/vendor: Stackable
-  name: opensearch-nodes-data-headless
-spec:
-  ports:
-  - name: http
-    port: 9200
-    protocol: TCP
-    targetPort: 9200
-  - name: transport
-    port: 9300
-    protocol: TCP
-    targetPort: 9300
-  publishNotReadyAddresses: true
-  selector:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/name: opensearch
-    app.kubernetes.io/role-group: data
-  type: ClusterIP
----
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/managed-by: opensearch.stackable.tech_opensearchcluster
-    app.kubernetes.io/name: opensearch
-    app.kubernetes.io/version: 3.0.0
-    stackable.tech/vendor: Stackable
-  name: opensearch
-  ownerReferences:
-  - apiVersion: opensearch.stackable.tech/v1alpha1
-    controller: true
-    kind: OpenSearchCluster
-    name: opensearch
-spec:
-  ports:
-  - name: http
-    port: 9200
-    protocol: TCP
-    targetPort: 9200
-  - name: transport
-    port: 9300
-    protocol: TCP
-    targetPort: 9300
-  publishNotReadyAddresses: true
-  selector:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/name: opensearch
-    stackable.tech/opensearch-role.cluster_manager: "true"
-  type: ClusterIP
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  labels:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/managed-by: opensearch.stackable.tech_opensearchcluster
-    app.kubernetes.io/name: opensearch
-    app.kubernetes.io/version: 3.0.0
-    stackable.tech/vendor: Stackable
-  name: opensearch-serviceaccount
-  ownerReferences:
-  - apiVersion: opensearch.stackable.tech/v1alpha1
-    controller: true
-    kind: OpenSearchCluster
-    name: opensearch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  labels:
-    app.kubernetes.io/component: nodes
-    app.kubernetes.io/instance: opensearch
-    app.kubernetes.io/managed-by: opensearch.stackable.tech_opensearchcluster
-    app.kubernetes.io/name: opensearch
-    app.kubernetes.io/version: 3.0.0
-    stackable.tech/vendor: Stackable
-  name: opensearch-rolebinding
-  ownerReferences:
-  - apiVersion: opensearch.stackable.tech/v1alpha1
-    controller: true
-    kind: OpenSearchCluster
-    name: opensearch
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: opensearch-clusterrole
-subjects:
-- kind: ServiceAccount
-  name: opensearch-serviceaccount
----
-apiVersion: policy/v1
-kind: PodDisruptionBudget
```

Comment:
Why did you remove all these assertions? We do not have another test case which checks if e.g. PodDisruptionBudgets are created.